.github/workflows/sdk.yml

```diff
@@ -39,9 +68,8 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: 1
-    - name: Build SDK
-      working-directory: sdk
-      run: go build -v ./...
-    - name: Test SDK
+    - name: Run SDK Integration Tests
       working-directory: sdk
-      run: go test -v ./...
+      run: for p in ./**/integrationtests; do go test -v $p; done
```

Comment:
`go test` will stream logs when using `-v` and testing a single package, which is why I'm finding the integration tests packages and then running go test on each one individually. Otherwise go test will batch the logs and show them at the end.